src/MicrosoftTeams.ts

```diff
@@ -409,6 +414,12 @@ namespace microsoftTeams
 
     export namespace authentication
     {
+        let authWindow: Window;
+        let authInterval: number;
+        let authParams: AuthenticateParameters;
+        handlers["authentication.authenticate.success"] = handleSuccess;
+        handlers["authentication.authenticate.failure"] = handleFailure;
+
```

Comment:
Don't we actually need to proxy *all* calls? For example if the child asks for the context